python/mxnet/kvstore.py

```diff
@@ -48,7 +48,7 @@ def updater_handle(key, lhs_handle, rhs_handle, _):
 
 class KVStore(object):
     """A key-value store for synchronization of values, over multiple devices."""
-    def __init__(self, handle):
+    def __init__(self, handle, name2idx=None):
```

Comment:
Yeah I am not sure if this is the best place to hold it. This is actually not required for the sparse feature. I can remove it temporarily for now.